protoutil/txutils.go

```diff
@@ -202,7 +203,8 @@ func createTx(
 		}
 
 		if !bytes.Equal(a1, r.Payload) {
-			return nil, errors.Errorf("ProposalResponsePayloads do not match with %v", r)
+			return nil, errors.Errorf("ProposalResponsePayloads do not match (base64):\n %s\n vs\n %s",
```

Comment:
let's remove the line feeds and (base64) and put the text in single quotes like:

ProposalResponsePayloads do not match : 'YmxhIGJsYQo=' vs 'YWxiIGFsYgo='

Reply:
I added base64 so that people know the output is in base64 and they can decode them. The decoded string has may text readable.

Reply:
ok sure :)